test/testdata/rewriter/minitest.rb.rewrite-tree.exp

```diff
@@ -14,11 +14,41 @@ class <emptyTree><<C <root>>> < ()
     end
 
     begin
-      ::T::Sig::WithoutRuntime.sig() do ||
-        <self>.params({}).void()
+      <emptyTree>::<C CONST> = ::T.unsafe(nil)
+      begin
+        ::T::Sig::WithoutRuntime.sig() do ||
+          <self>.params({}).void()
+        end
+        def <test_allows constants inside of IT><<C <todo sym>>>(&<blk>)
+          ::Module.const_set(:"CONST", 10)
+        end
+      end
+    end
```

Comment:
This shows that typical constant assignments get pulled out of method definitions and replaced with a `const_set`.